### PR TITLE
French translation for the `color-scheme` CSS property reference.

### PR DESCRIPTION
**Pull request by audrasjb**
This PR aims to create a French translation for the `color-scheme` CSS property reference.

### CODE REVIEW

File: files/fr/web/css/color-scheme/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: color-scheme
+slug: Web/CSS/color-scheme
+browser-compat: css.properties.color-scheme
+translation_of: Web/CSS/color-scheme
+---
+<div>{{CSSRef}}</div>
+
+<p>La propriété CSS <strong><code>color-scheme</code></strong> permet d'indiquer les jeux de couleurs dans lesquel un élément peut être rendu sans risque.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/color-scheme/index.html:9]: typo "lesquel" > "lesquelles"

File: files/fr/web/css/color-scheme/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<p>Les jeux de couleurs communément utilisés par les systèmes d'exploitation sont "light" (clair) et "dark" (sombre), ou "day mode" (mode jour) et "night mode" (mode nuit). Lorsqu'un utilisateur sélectionne un de ces jeux de couleurs, le système d'exploitation fait des ajustements à l'interface utilisateur. Cela comprend les contrôles de formulaires, les barres de défilement et les valeurs utilisées par le système de couleurs CSS.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/color-scheme/index.html:10]: (not blocking/mandatory: I'd use `<i>`s to wrap English terms.)

File: files/fr/web/css/color-scheme/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+<h2 id="Formal_syntax">Syntaxe formelle</h2>
+
+<p>{{csssyntax}}</p>
+
+<h2 id="Examples">Exemples</h2>
+
+<h3 id="Adapting_to_color_schemes">Adaptation aux jeux de couleurs</h3>
+
+<p>Pour activer la préférence de jeu de couleurs de l'utilisateur sur l'ensemble de la page, <code>color-scheme</code> on peut être déclaré sur l'élément {{cssxref(":root")}}.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/color-scheme/index.html:50]: -on

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Minor changes left :) Thanks already @audrasjb

File: files/fr/web/css/color-scheme/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html:9]: Maybe we should rather use "lesquels" given "jeux" is masculine? 🤔

Comment (SphinxKnight) [reply files/fr/web/css/color-scheme/index.html:9]: OK for "lesquels"👌

File: files/fr/web/css/color-scheme/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html:10]: LGTM

File: files/fr/web/css/color-scheme/index.html (hunk 3 above)
Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html:50]: yep

Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html]: fixed in https://github.com/mdn/translated-content/pull/1708/commits/cc2a98d61a693a04fa09a0d4bbfff1b6b2d36346

File: files/fr/web/css/color-scheme/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html]: fixed in https://github.com/mdn/translated-content/pull/1708/commits/cc2a98d61a693a04fa09a0d4bbfff1b6b2d36346

**Review by audrasjb — COMMENTED** [review]
ffffff [edit: sorry, wrong manip]

File: files/fr/web/css/color-scheme/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+<h3 id="Values">Valeurs</h3>
+
+<dl>
+	<dt><code>normal</code></dt>
+	<dd>Indique que l'élément ne tient pas compte d'un éventuel jeu de couleurs et qu'il devrait être rendu à l'aide du jeu de couleurs par défaut du navigateur.</dd>
+	<dt><code>light</code></dt>
+  <dd>Indique que l'élément peut être rendu en utilisant le jeu de couleurs claires du système d'exploitation.</dd>
+  <dt><code>dark</code></dt>
+	<dd>Indique que l'élément peut être rendu en utilisant le jeu de couleurs sombres du système d'exploitation.</dd>
+</dl>
```
[the remark this answers is not in the document]
Comment (audrasjb) [reply files/fr/web/css/color-scheme/index.html]: fixed in https://github.com/mdn/translated-content/pull/1708/commits/cc2a98d61a693a04fa09a0d4bbfff1b6b2d36346

File: files/fr/web/css/color-scheme/index.html (hunk 1 above)
Comment (SphinxKnight) [reply files/fr/web/css/color-scheme/index.html:9]: Il semble que celle-ci est restée (s à la fin) :x

Comment (SphinxKnight) [reply files/fr/web/css/color-scheme/index.html]: Fixed w/ 4869520bd9d6988ecdc192334ea8e2cf40e7f3a9

**Review by SphinxKnight — APPROVED** [review]
I've fixed the last trivial typo, let's not get complicated.

**Pull request merged**